src/main/resources/avro/variantmethods.avdl

```diff
@@ -148,68 +148,3 @@ org.ga4gh.models.Variant getVariant(
   The ID of the `Variant`.
   */
   string id) throws GAException;
-
-/******************  /callsets/search  *********************/
```

Comment:
There will be the following in metadatamethods.avdl (which doesn't exist right now):
```java
record SearchAnalysisSamplesRequest {
  string datasetId; // not null
  array<string> sampleNames = [];
}
record SearchAnalysisSamplesResponse {
  array<string> analysisSampleIds;
}
```